### DISCUSSION TITLE
Linking multiple Nightscout instances

### DISCUSSION THREAD

**Discussion by edpomfret** [nielsmaerten/gluco-check#183] [Q&A]
I've got Gluco check working great with Google assistant for my blood sugars,  but is it possible to set up an additional routine for my son's. He has the same set up as me but obviously a different Nightscout site. But I can't work out how to get Google assistant to check the alternative Nightscout site reading. Any tips?

**Comment by nielsmaerten** [comment]
Hey, sorry it took me a while to get back to you. Had to check the code to make sure.
I'm afraid this is not currently possible. The way Gluco Check is programmed, 1 Google-account can only access 1 Nightscout site. 
(It's a 1-to-1 link.)

If you both have your own Nightscout instance, then it should be possible to link each one to that person's Google account respectively. Then if you have a Google Home that you're both signed into, you and your son should be able to ask Gluco Check for your own values, but not the other person's.

Making this work would require some parts of Gluco Check to be rewritten. It's not impossible, but as this is a free project I'm afraid I don't have the resources right now.
